### ISSUE TITLE
crlfmt strips comments

### ISSUE THREAD

**Issue by andreimatei** [cockroachdb/crlfmt#6]
```
/Users/andrei/work/src/github.com/cockroachdb/cockroach/storage/engine/mvcc.go:476
-func MVCCPutProto(
-   ctx context.Context,
-   engine ReadWriter,
-   ms *enginepb.MVCCStats, // can be nil is the key is unreplicated and doesn't affect stats
-   key roachpb.Key,
-   timestamp hlc.Timestamp,
-   txn *roachpb.Transaction,
-   msg proto.Message,
-) error {
+func MVCCPutProto(
+   ctx context.Context,
+   engine ReadWriter,
+   ms *enginepb.MVCCStats,
+   key roachpb.Key,
+   timestamp hlc.Timestamp,
+   txn *roachpb.Transaction,
+   msg proto.Message,
+) error {
```

**Comment by tamird** [comment]
Seems like an unconventional place to put a comment.

**Comment by RaduBerinde** [comment]
The fix for #5 could apply to this as well: first check if the existing code is correctly formatted, and if so don't change it. The code can also be conservative and just don't reformat something which contains comments or other extraneous stuff.

**Comment by bdarnell** [comment]
Looks like a perfectly reasonable place for a comment to me. In fact, allowing for per-parameter inline comments seems like one of the advantages of the one-parameter-per-line style.
